File: C++/searching-inside-the-sequence-1.cpp
#include <iostream>  

using namespace std;  

int main() {  
    int n, q;  
    cin >> n >> q;  
    
    int a[n];
    for(int i = 0; i < n; i++) {  
        cin >> a[i];
    }  
 
    for(int i = 0; i < q; i++) {  
        int x;  
        cin >> x; 
        
        int count = 0;
        for(int item : a) {  
            if(item < x) {  
                count++;
            }  
        }  
        cout << count << "\n";
    }  
    return 0;  
}  

File: Python/searching-inside-the-sequence-1.py
n, q = map(int, input().split(" "))
a = list(map(int, input().split(" ")))
for i in range(q):
    x = int(input())
    count = 0
    for item in a:
        if item < x:
            count += 1
    print(count, end="\n")